### DISCUSSION TITLE
[Kavita] Customizable Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by DieselTech** [Kareadita/Kavita#2563] [Ideas]
### Idea Description

Posted by John Atkins
· Dec 2021
· 11 Upvotes
https://feats.kavitareader.com/posts/25/customizable-keyboard-shortcuts

Description

Requesting additional keyboard shortcuts, preferably being able to customize them.

For example, on ComiRack there's a list of customizable keyboard shortcuts: https://i.imgur.com/vicsTN6.png

In particular, W = scroll up, S = Scroll down, A = Previous Chapter, D = Next Chapter.

### Duration of Using Kavita

42

### Idea Category

Feature Enhancement

### Agreement

- [X] I agree that this is solely for submitting ideas, and I will search for existing ideas before posting.

**Comment by CraigAstillRVU** [comment]
To add another vote for customisable shortcuts: On a Mac, CMD+f to search within an open epub in the browser results in Kavita toggling into FullScreen mode.

**Comment by bloksma** [comment]
I would also suggest to shortcuts for zooming (width override) and image scaling. But the best would be to have shortcuts for every setting in the reader.

I've used CdisplayEX a lot, and I miss the keyboard navigation.

**Comment by majora2007** [comment]
This is now in the nightly release (v0.8.8.5) and will be delivered in v0.8.9.
